src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/rgw/rgw-daemon-list/rgw-daemon-list.component.html

```diff
@@ -0,0 +1,12 @@
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item">Object Gateway</li>
+    <li class="breadcrumb-item active">Daemons</li>
```

Comment:
"Daemons" is not a sub-menu entry of "Object Gateway" menu entry, so IMO it's not correct to include "Daemons" on the breadcrumb.

Reply:
Done